src/components/menu-top/MenuTop.vue

```diff
@@ -68,6 +68,27 @@
             </b-checkbox>
           </b-navbar-item>
         </b-navbar-dropdown>
+
+        <b-navbar-dropdown label="Grid" data-test="menu-top--grid">
+          <b-navbar-item
+            data-test="menu-top--grid-one"
+            @click="changeStepGrid(1)"
+          >
+            One Step
+          </b-navbar-item>
+          <b-navbar-item
+            data-test="menu-top--grid-two"
+            @click="changeStepGrid(2)"
+          >
+            Two Step
+          </b-navbar-item>
+          <b-navbar-item
+            data-test="menu-top--grid-four"
+            @click="changeStepGrid(4)"
+          >
+            Four Step
+          </b-navbar-item>
+        </b-navbar-dropdown>
```

Comment:
This works okay, but I don't think this should be it's own top navbar item. We should reserve creating a new dropdown at the top level for new categories (For example, see the top menu of any program you use, "File", "Edit", "View", "Help", etc.)

Let's consider other options... How about the bottom menu next to all the tool icons?

Reply:
I wonder if there is a way to get some tiered/tree menus in Buefy. I'm thinking something like the Format -> Number selection in google sheets. That way we can put it in the view section...

<img width="839" alt="image" src="https://user-images.githubusercontent.com/29964426/123559925-fa81cc80-d753-11eb-8140-b89f462e2954.png">